recipe/install_nvcc.sh

```diff
@@ -53,17 +63,34 @@ then
     return 1
 fi
 
-if [[ \$(grep -q "CUDA Version ${PKG_VERSION}" \${CUDA_HOME}/version.txt) -ne 0 ]]
+if [[ -z "\$(\${CUDA_HOME}/bin/nvcc --version | grep "Cuda compilation tools, release ${PKG_VERSION}")" ]]
 then
-    echo "Version of installed CUDA didn't match package"
-    return 1
+  echo "Version of installed CUDA didn't match package"
+  return 1
 fi
 
 export CUDA_HOME="\${CUDA_HOME}"
 export CFLAGS="\${CFLAGS} -I\${CUDA_HOME}/include"
 export CPPFLAGS="\${CPPFLAGS} -I\${CUDA_HOME}/include"
 export CXXFLAGS="\${CXXFLAGS} -I\${CUDA_HOME}/include"
 
+### CMake configurations
+
+# CMake looks up components in CUDA_PATH, not CUDA_HOME
+export CUDA_PATH="\${CUDA_HOME}"
+# New-style CUDA integrations in CMake
+CMAKE_ARGS="\${CMAKE_ARGS:-} -DCUDAToolkit_ROOT=\${CUDA_HOME}"
+# Old-style CUDA integrations in CMake
+## See https://github.com/conda-forge/nvcc-feedstock/pull/58#issuecomment-752179349
+CMAKE_ARGS+=" -DCUDA_TOOLKIT_ROOT_DIR=\${CUDA_HOME}"
+## Prevent errors like https://github.com/floydhub/dl-docker/issues/59
+CMAKE_ARGS+=" -DCMAKE_LIBRARY_PATH=\${CUDA_HOME}/lib64/stubs"
+## Avoid https://github.com/conda-forge/openmm-feedstock/pull/44#issuecomment-753560234
+CMAKE_ARGS+=" -DCMAKE_FIND_ROOT_PATH_MODE_INCLUDE=BOTH"
```

Comment:
Can you see if adding `$CUDA_HOME` to `CMAKE_PREFIX_PATH` env var works?

Reply:
It does not work, at least for OpenMM.

Reply:
Can you try fixing `CMAKE_FIND_ROOT_PATH`? It's currently https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/master/recipe/activate-gcc.sh#L142

Reply:
I am trying by doing this:

```
CMAKE_ARGS="$(echo ${CMAKE_ARGS} | sed -E -e "s|(-DCMAKE_FIND_ROOT_PATH=\S+)|\1;$CUDA_HOME|")"
```

Which seems to work, but with warnings:

```
-- Found CUDA: /usr/local/cuda (found version "11.0") 
CMake Warning (dev) at /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/_build_env/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (OPENCL)
  does not match the name of the calling package (OpenCL).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake_modules/FindOpenCL.cmake:86 (find_package_handle_standard_args)
  CMakeLists.txt:338 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found OPENCL: /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/libOpenCL.so  
-- Configuring done
CMake Warning at plugins/cudacompiler/CMakeLists.txt:71 (ADD_LIBRARY):
  Cannot generate a safe runtime search path for target OpenMMCudaCompiler
  because files in some directories may conflict with libraries in implicit
  directories:

    runtime library [libnvrtc.so.11.0] in /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib may be hidden by files in:
      /usr/local/cuda/lib64

  Some of these libraries may not be found correctly.
```

Reply:
Try adding `CUDA_HOME/include` instead of `CUDA_HOME`

Reply:
That takes us back to square one:

```
-- Configuring done
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
CUDA_TOOLKIT_INCLUDE (ADVANCED)
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler
   used as include directory in directory /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/work/plugins/cudacompiler

CMake Error in plugins/cudacompiler/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "OpenMMCudaCompiler":

    "CUDA_TOOLKIT_INCLUDE-NOTFOUND"



CMake Error in plugins/cudacompiler/CMakeLists.txt:
  Found relative path while evaluating include directories of
  "OpenMMCudaCompiler":

    "CUDA_TOOLKIT_INCLUDE-NOTFOUND"
```

Reply:
Can you try adding CUDA_HOME to the front?

Reply:
That seems to work. With:

```
CMAKE_ARGS="$(echo ${CMAKE_ARGS} | sed -E -e "s|(-DCMAKE_FIND_ROOT_PATH=)(\S+)|\1$CUDA_HOME;\2|")"
```

We get:

```
-- Found CUDA: /usr/local/cuda (found version "11.0") 
CMake Warning (dev) at /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/_build_env/share/cmake-3.19/Modules/FindPackageHandleStandardArgs.cmake:424 (message):
  The package name passed to `find_package_handle_standard_args` (OPENCL)
  does not match the name of the calling package (OpenCL).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake_modules/FindOpenCL.cmake:86 (find_package_handle_standard_args)
  CMakeLists.txt:338 (FIND_PACKAGE)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found OPENCL: /home/conda/feedstock_root/build_artifacts/openmm_1609778103608/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_plac/lib/libOpenCL.so  
-- Configuring done
-- Generating done
```

Reply:
See last commits for this addition.